src/components/panels/Temperature/TemperaturePanelListItemNevermoreValue.vue

```diff
@@ -0,0 +1,153 @@
+        const value = this.printerObject[this.keyName] ?? null
+        if (isNaN(value)) return null
+
+        return value
+    }
+
+    get intake_value(): number | null {
+        const name = `intake_${this.keyName}`
+
+        return this.printerObject[name] ?? null
+    }
+
+    get intake_value_min(): number | null {
+        const name = `intake_${this.keyName}_min`
+
+        return this.printerObject[name] ?? null
+    }
+
+    get intake_value_max(): number | null {
+        const name = `intake_${this.keyName}_max`
+
+        return this.printerObject[name] ?? null
+    }
+
+    get exhaust_value(): number | null {
+        const name = `exhaust_${this.keyName}`
+
+        return this.printerObject[name] ?? null
+    }
+
+    get exhaust_value_min(): number | null {
+        const name = `exhaust_${this.keyName}_min`
+
+        return this.printerObject[name] ?? null
+    }
+
+    get exhaust_value_max(): number | null {
+        const name = `exhaust_${this.keyName}_max`
+
+        return this.printerObject[name] ?? null
+    }
+
+    get unit(): string | null {
+        switch (this.keyName) {
+            case 'temperature':
+                return '°C'
+            case 'pressure':
+                return 'hPa'
+            case 'humidity':
+                return '%'
+        }
+
+        return null
+    }
+
+    get digits() {
+        return ['gas', 'pressure'].includes(this.keyName) ? 0 : 1
+    }
+
+    get formatValue() {
+        let intake_value = this.intake_value?.toFixed(this.digits)
+        let exhaust_value = this.exhaust_value?.toFixed(this.digits)
+        if (this.intake_value === null) intake_value = '--'
+        if (this.exhaust_value === null) exhaust_value = '--'
+
+        // return only the value, if unit is null
+        if (this.unit === null) return `${intake_value} > ${exhaust_value}`
+
+        return `${intake_value} ${this.unit} > ${exhaust_value} ${this.unit}`
+    }
+
+    get formatValue_min() {
+        let intake_value = this.intake_value_min?.toFixed(this.digits)
+        let exhaust_value = this.exhaust_value_min?.toFixed(this.digits)
+        if (this.intake_value === null) intake_value = '--'
+        if (this.exhaust_value === null) exhaust_value = '--'
+
+        // return only the value, if unit is null
+        if (this.unit === null) return `${intake_value} > ${exhaust_value}`
+
+        return `${intake_value} ${this.unit} > ${exhaust_value} ${this.unit}`
+    }
+
+    get formatValue_max() {
+        let intake_value = this.intake_value_max?.toFixed(this.digits)
+        let exhaust_value = this.exhaust_value_max?.toFixed(this.digits)
+        if (this.intake_value === null) intake_value = '--'
+        if (this.exhaust_value === null) exhaust_value = '--'
+
+        // return only the value, if unit is null
+        if (this.unit === null) return `${intake_value} > ${exhaust_value}`
+
+        return `${intake_value} ${this.unit} > ${exhaust_value} ${this.unit}`
+    }
```

Comment:
duplicated code. could be merged into a single parameterized function.

```suggestion
    get formatValue() {
        return getFormatedValue(this.intake_value, this.exhaust_value)
    }

    get formatValue_min() {        
        return getFormatedValue(this.intake_value_min, this.exhaust_value_min)
    }

    get formatValue_max() {   
        return getFormatedValue(this.intake_value_max, this.exhaust_value_max)
    }

    
    getFormatedValue(intake: number | null, exhaust: number | null): string {
        let intake_value = intake?.toFixed(this.digits)
        let exhaust_value = exhaust?.toFixed(this.digits)
        if (this.intake_value === null) intake_value = '--'
        if (this.exhaust_value === null) exhaust_value = '--'

        // return only the value, if unit is null
        if (this.unit === null) return `${intake_value} > ${exhaust_value}`

        return `${intake_value} ${this.unit} > ${exhaust_value} ${this.unit}`
    }
```